code/datums/supplypacks.dm

```diff
@@ -462,6 +462,27 @@ WEAPONS
 	containername = "\improper explosive HEDP grenade crate (WARNING)"
 	group = "Weapons"
 
+/datum/supply_packs/explosives_hidp
+	name = "M40 HIDP incendary grenade box crate (x25)"
```

Comment:
```suggestion
 	name = "M40 HIDP incendiary grenade box crate (x25)"
```

Reply:
Ok I'll fix it, thanks for the observation